pkgs/development/mobile/xcodeenv/compose-xcodewrapper.nix

```diff
@@ -8,6 +8,8 @@ assert stdenv.isDarwin;
 stdenv.mkDerivation {
   pname = "xcode-wrapper${lib.optionalString allowHigher "-plus"}";
   inherit version;
+  # Fix 'xcodebuild: Operation not permitted' when 'sandbox=relaxed' is used.
```

Comment:
I would expect a comment like this:

https://github.com/NixOS/nixpkgs/blob/26dfd6de3fbb67a78572957a91fc5ede000b14b2/pkgs/development/haskell-modules/generic-stack-builder.nix#L25-L26

Reply:
Sure, that's a good point.

Reply:
Done.